public/about.html

```diff
@@ -33,43 +33,260 @@
+          <ul>
+            <li>Risk Communication</li>
+            <li>Public Engagement</li>
+            <li>Procurement Reports</li>
+            <li>Financial Reports	</li>
+          </ul>
+        </div>
+      </div>
+      <hr>
+    </div>
+
+    <div class="legal_response">
+      <div class="legal_response_content">
+        <div class="legal_response_subcontent">
+          <h4>LEGAL RESPONSE</h4>
+          <p>
+            Gamit ang Bantay Bayan Helpdesk, maaaring mag-sumite ng tanong o
+            reklamo ang bawat mamamayan ukol sa pag-responde ng LGU sa COVID-19. Ito ay tinutugunan ng mga
+            Bantay Bayan volunteer na may kaalaman at karanasan sa mga legal na proseso. 
+          </p>
+        </div>
+        <div class="legal_response_logo">
+          <img src="../assets/images/legal_response_logo1.png">
+          <img src="../assets/images/legal_response_logo2.png" class="lr_logo2">
+        </div>
+      </div>
+      <hr>
+    </div>
+
+    <div class="usap_tayo">
+      <div class="usap_tayo_content">
+        <div class="usap_tayo_subcontent">
+          <h4>USAP TAYO!</h4>
+          <p>
+            Inilunsad din ang Usap Tayo!, mga online forum na layong palalimin ang diskurso at kaalaman ng
+            mga volunteer at kapwa kabataan patungkol sa iba’t ibang isyung panlipunan na maaaring tugunan 
+            ng mabuting pamamahala. Kabilang sa usapan ang citizen and youth empowerment, local governance, 
+            economics of social protection, public health, women empowerment, human rights, media and
+            journalism, at labor rights. Maaaring pakinggan ang mga session sa Spotify at Anchor!
+          </p>
+        </div>
+        <div class="usap_tayo_logo">
+          <img src="../assets/images/usap_tayo_logo.png">
+        </div>
+      </div>
+      <iframe src="https://open.spotify.com/embed-podcast/show/5F2Yxmz0AmVONr1h34rvBV" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>
+    </div>
+  </div>
+      
+  <div class="partner_details">
+    <div class="partner_label">
+      <p>Ang Bantay Bayan ay hatid sa inyo ng mga sumusunod na organisasyon:</p>
+    </div>
+    <div class="partner_content">
+      <div class="partner_subcontent1">
+        <div class="partner_subcontent1_head">
+          <img src="../assets/images/good_gov_logo.png" class="abt_good_gov_logo">
+          <h4>GoodGovPH</h4>
+          <a href="https://facebook.com/BantayBayanTayo" target="_blank"><img src="../assets/images/about_fb.png" class="abt_fb_1"></a>
+          <a href="https://l.facebook.com/l.php?u=http%3A%2F%2Ftwitter.com%2FBantayBayanTayo%3Ffbclid%3DIwAR1XGVB-ntc44bMnby-eIy2M2PxsyXfxhafhCHt-gfrmQFuWewhdFv_Bzi8&h=AT2VaHz6xjRRLGQ4U1AhXaaHJ1joIIB_UgjrqD2nqNQj33cuhgoLPO34STyFcbMJFNg8pBQquCteBqRsFzSUqBegPJ18J-jDkgWkZsokoHSfS-1W8ELTeOxdE55cVKHi7qmbi2_IKpInrIvGfRfeSg" target="_blank"><img src="../assets/images/about_twitter.png" class="abt_twitter_1"></a>
```

Comment:
`rel="noopener noreferrer"` should be added to links containing target="_blank" as a precaution against reverse tabnabbing. For more information, please refer to the following article:
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Link is is supposed to be https://twitter.com/GoodGovPH